utils/dwarf.c

```diff
@@ -1624,9 +1634,16 @@ static char *get_base_comp_dir(struct rb_root *dirs)
 	return max->name;
 }
 
-static void build_dwarf_info(struct uftrace_dbg_info *dinfo, struct uftrace_symtab *symtab,
-			     enum uftrace_pattern_type ptype, struct strv *args, struct strv *rets)
+static int build_dwarf_info_cb(Dwfl_Module *mod __attribute__((unused)),
+			       void **userdata __attribute__((unused)), const char *name,
```

Comment:
uftrace uses `-Wno-unused-parameter`, I think the `unused` attributes can go away.